plugins/module_utils/utils.py

```diff
@@ -26,3 +26,29 @@ def generate_api_endpoint(path, **kwargs):
     # return full_url
 
     return path if not kwargs else "{0}?{1}".format(path, "&".join(["{0}={1}".format(key, value) for key, value in kwargs.items()]))
+
+
+def format_list_dict(list_dict, keys_map):
+    """
+    Convert an Python list of dictionaries into its equivalent NDO API format.
+    All keys must be definied in the keys map even if no conversion is needed for some keys.
```

Comment:
-> All keys must be defined in the keys map even if no conversion is needed for some keys.